app/lib/widgets/cards/productCard/shared/left_top_icon.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:flex_ui/tokens/colors.dart';
+import 'package:flex_ui/tokens/sizes.dart';
+import 'package:flutter/material.dart';
+
+class LeftTopIcon extends StatelessWidget {
+  const LeftTopIcon({
+    super.key,
+    required this.label,
+    this.iconBackgroundColor = FlexColors.secondary,
+    this.textColor = Colors.white,
```

Comment:
If we're going to use FlexColors and not ThemeData, we can use `FlexColors.onSecondary` here

Reply:
But onSecondary is for action color, like a tap, isn't it ? Ok it's not a big deal but we need a ticket to manage all the AppTheme / FlexColors

Reply:
The goal of `secondary` and `onSecondary` is background color and foreground color. Yes, I'll write up a ticket so we can manage the Theme questions we have.